lib/event-service.js

```diff
@@ -0,0 +1,59 @@
+var EVENTSERVICE_SERVICE_UUID          = 'E95D93AF251D470AA062FA1922DFA9A8'.toLowerCase();
+var MICROBITEVENT_CHARACTERISTIC_UUID = 'E95D9775251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var CLIENTREQUIREMENTS_CHARACTERISTIC_UUID = 'E95D23C4251D470AA062FA1922DFA9A8'.toLowerCase();
+var CLIENTEVENT_CHARACTERISTIC_UUID = 'E95D5404251D470AA062FA1922DFA9A8'.toLowerCase();
+
+var microbit_event_subscribed = 0;

```

Comment:
Whatever you say boss ;-)

My this is hard work. Had I known..... :-)

I'll get to all this stuff at the weekend hopefully.